static/js/protein/chart.ts

```diff
@@ -29,10 +29,61 @@ export function drawTissueScoreChart(
   id: string,
   data: { name: string; value: string }[]
 ): void {
-  // TODO: convert data to tmpData format.
-  const tmpData = [{ x1: 20, x2: 60, y1: 30, y2: 50 }];
+  const tmpData = [
+    { name: "Cartilage", value: 3 },
+    { name: "Eye", value: 2 },
+    { name: "Liver", value: 1 },
+    { name: "Skin", value: 0 },
+  ];
+  const testData = data;
+  function convertData(val) {
+    if (val == "ProteinExpressionNotDetected") {
+      return 0;
+    } else if (val == "ProteinExpressionLow") {
+      return 1;
+    } else if (val == "ProteinExpressionMedium") {
+      return 2;
+    } else if (val == "ProteinExpressionHigh") {
+      return 3;
+    }
+  }
+  const reformat = data.map((item) => {
+    const newObj: any = {};
```

Comment:
usually we don't want to use `any` as the data type. Maybe create a new interface for {name: string, value: number}

Reply:
Added interface !